### PR TITLE
Easily change between Float32 and Float64.

### PR DESCRIPTION
**Pull request by prabhuramachandran**
This just makes it easy to change between float32 and float64 to test performance.

### CODE REVIEW

**Review by gryznar — COMMENTED** [review]
I do not think, that this is step in the right direction

**Comment by prabhuramachandran** [comment]
> I do not think, that this is step in the right direction

It would help if you can explain why you think so and what you want to see changed? The current example shows fp32 performance, many scientific codes use double precision and this small change allows one to quickly check this. Moreover, the `type` alias already existed.

**Comment by gryznar** [comment]
I agree with you, but previous version is much more readable. Most of users just want to run this as it is. Familiarizing with such code, where types are aliased to not meaningful value is harder than with code in which types are explicit. If you would like to play with it, you can just set it locally

**Comment by prabhuramachandran** [comment]
> I agree with you, but previous version is much more readable. Most of users just want to run this as it is. Familiarizing with such code, where types are aliased to not meaningful value is harder than with code in which types are explicit. If you would like to play with it, you can just set it locally

Thanks for the elaboration. This is why I did not send a more complex solution, the idea was to just do the minimum, the only change here is the alias called `Float` (other names would also work), `type` was already there and this makes it easy for someone to change two lines to play with the performance vs. changing the type in many places.

**Comment by prabhuramachandran** [comment]
@gryznar -- I have modified the code to not introduce any new aliases, this should not change the readability and still makes it easy to change the data type with a single line.

File: examples/matmul.mojo [hunk 1]
```diff
@@ -29,7 +29,7 @@ alias M = 512
 alias N = 512
 alias K = 4096
 alias type = DType.float32
-
+# alias type = DType.float64
```
[the remark this answers is not in the document]
Comment (prabhuramachandran) [reply examples/matmul.mojo]: Thank you, I've made the change.

**Review by jackos — APPROVED** [review]

**Pull request merged**